### PR TITLE
[v7r3] docs: moved docs of ComputingElement to dirac.cfg example

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*docs
CHANGE: moved docs of ComputingElement from ConfReference to dirac.cfg

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
```
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Resources/computingelements.rst:103: WARNING: undefined label: resourcescomputing
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/ServerInstallations/scalingAndLimitations.rst:144: WARNING: undefined label: res-comp-htcondor
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/WorkloadManagement/JobsMatching.rst:13: WARNING: undefined label: resourcescomputing
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/index.rst:151: WARNING: undefined label: resourcescomputing
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/WorkloadManagement/index.rst:39: WARNING: undefined label: resourcescomputing
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/src/DIRAC/Resources/Computing/ARCComputingElement.py:docstring of DIRAC.Resources.Computing.ARCComputingElement:6: WARNING: undefined label: resourcescomputing
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py:docstring of DIRAC.Resources.Computing.HTCondorCEComputingElement:7: WARNING: undefined label: resourcescomputing
```

File: dirac.cfg [hunk 1]
```diff
@@ -419,6 +419,94 @@ Systems
...
+
+      # or adding some generic pilot options (only for pilots submitted by SiteDirectors)
+      # the example below will add the environment variable DIRACSYSCONFIG (see :ref:`bashrc_variables`)
+      ExtraPilotOptions = --userEnvVariables DIRACSYSCONFIG:::pilot.cfg
+
+      # for adding the --modules=value option to dirac-pilot
+      Modules =
+
+      # for adding the --pipInstallOptions=value to dirac-pilot
+      PipInstallOptions = --index-url https://lhcb-repository.web.cern.ch/repository/pypi/simple
```
Comment (chrisburr) [inline dirac.cfg:453]: @atsareg I'm pretty sure you can use this option to add COMDIRAC to Python 3 pilots

**Review by chrisburr — DISMISSED** [review]

File: docs/source/AdministratorGuide/Resources/computingelements.rst [hunk 2]
```diff
@@ -99,8 +99,7 @@ of the *ComputingElement* is located in the inside the corresponding site sectio
 This is the general structure in which specific CE descriptions are inserted.
 The CE configuration is part of the general DIRAC configuration
 It can be placed in the general Configuration Service or in the local configuration of the DIRAC installation.
-
-Additional info can be found :ref:`here <resourcesComputing>`.
+Examples of the configuration can be found in the `Full Configuration Example`_, in the *Resources/Computing* section.
```
Comment (fstagni) [inline docs/source/AdministratorGuide/Resources/computingelements.rst:102]: @andresailer is it this the reason why the documentation build fails? I can't see the actual error.

Comment (andresailer) [reply docs/source/AdministratorGuide/Resources/computingelements.rst:102]: Yes, the error is in the logs
```
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Resources/computingelements.rst:99: ERROR: Unknown target name: "full configuration example".
```
You need to scroll to the right to see it 
This needs the sphinx.ext.autosectionlabel extension to be enabled? 
https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html#module-sphinx.ext.autosectionlabel

Comment (fstagni) [reply docs/source/AdministratorGuide/Resources/computingelements.rst:102]: No idea. How would we try?

Comment (andresailer) [reply docs/source/AdministratorGuide/Resources/computingelements.rst:102]: add the extension to the list
https://github.com/DIRACGrid/DIRAC/blob/0696d01e08f417cdde1c877fec40e4122a2642d7/docs/source/conf.py#L104-L108

Comment (fstagni) [reply docs/source/AdministratorGuide/Resources/computingelements.rst:102]: It did not work

Comment (andresailer) [inline docs/source/AdministratorGuide/Resources/computingelements.rst:102]:
```suggestion
Examples of the configuration can be found in the :ref:`Full Configuration Example`, in the *Resources/Computing* section.
```

Comment (andresailer) [reply docs/source/AdministratorGuide/Resources/computingelements.rst]: This extension causes a lot of warnings by itself, so better manually add the label for the Full Configuration Example and refer to that, via the script that creates that file.

Comment (andresailer) [reply docs/source/AdministratorGuide/Resources/computingelements.rst]:
```
[snip]
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/FTS3Agent.rst:2: WARNING: duplicate label fts3agent, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/DataManagement/fts3.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/MoveReplica.rst:2: WARNING: duplicate label movereplica, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/PutAndRegister.rst:2: WARNING: duplicate label putandregister, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RegisterFile.rst:2: WARNING: duplicate label registerfile, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RegisterReplica.rst:2: WARNING: duplicate label registerreplica, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RemoveFile.rst:2: WARNING: duplicate label removefile, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RemoveReplica.rst:2: WARNING: duplicate label removereplica, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.rst:2: WARNING: duplicate label replicateandregister, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RequestOperations_Module.rst:10: WARNING: duplicate label modules, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/Agent_Module.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Client/Client_Module.rst:2: WARNING: duplicate label client, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/Core/Tornado/Client/Client_Module.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Client/Client_Module.rst:10: WARNING: duplicate label subpackages, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/Agent_Module.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Client/Client_Module.rst:18: WARNING: duplicate label modules, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RequestOperations_Module.rst
[snip]
```

**Comment by fstagni** [comment]
Documentation is building correctly now @andresailer

**Review by andresailer — APPROVED** [review]
Thanks! sorry for the hassle with the link!

**Pull request closed without merging**